packages/components/src/color-palette/index.native.js

```diff
@@ -260,6 +260,7 @@ function ColorPalette( {
 									selected: isSelected( color ),
 								} }
 								accessibilityHint={ color }
+								accessibilityLabel={ color }
```

Comment:
I posit we should consider a more helpful accessibility label than the HEX value. From my experience, stepping through buttons with this label is a bit difficult to follow. Is there a color label or slug that could be used in place of the raw value? E.g. `light purple` or `light-purple`?

Reply:
There is! Good catch, I'll push a change to this branch in a bit. Thanks for the suggestion!

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/49917/commits/6e8f955c394384de4d1f29684ac12408b30d35ee